src/redux/migrations.ts

```diff
@@ -41,6 +40,16 @@ export function updateCachedQuoteParams(cachedQuoteParams: {
 
 const DEFAULT_DAILY_PAYMENT_LIMIT_CUSD_LEGACY = 1000
 
+const exchangeInitialState = {
```

Comment:
Could we just export this instead of remaking it in `src/redux/migrations.test.ts`?